src/driver/odoc_units_of.mli

```diff
@@ -1,6 +1,6 @@
 open Odoc_unit
 
-type indices_style = Voodoo | Normal | Automatic
+type indices_style = Voodoo | Normal of Fpath.t option | Automatic
```

Comment:
It would be nice to document Normal's arguments, or perhaps use an inline record.